assets/js/source.js

```diff
@@ -161,20 +162,23 @@ export async function switchDateFormat() {
 }
 
 function resetScrollTracker() {
-  let window_inner_height = window.innerHeight
-  let window_offset = window.pageYOffset
-  let client_height = document.body.clientHeight
-  let subhead = document.querySelector("div.subhead")
-  let header = document.querySelector("nav")
-  let nav_height = subhead.offsetHeight + header.offsetHeight
-
-  // even if we're close to the bottom, we're at the bottom (for mobile browsers)
-  // should make this dynamic
-  if (window_inner_height + window_offset - nav_height >= client_height - 129) {
-    window.scrollTracker = true
-  } else {
-    window.scrollTracker = false
-  }
+  const observer =
+      new IntersectionObserver((entries, observer) => {
+        entries.forEach((entry) => {
+          let searchInView = entry.isIntersecting
+          if (searchInView) {
+            // stick to bottom
+            
+            window.scrollTracker = true
+          } else {
+            // don't stick to bottom
+            window.scrollTracker = false
+          }
+        })
+      })
+    
+    const target = document.querySelector("#search-tip")
```

Comment:
could we not use the search tip as the target? I'm thinking of removing it and moving all the tips to the docs so that there is better visibility for them.

Reply:
What's more visible than actually in the UI?

Reply:
I changed the observer target tho.